test/service/test_manifest.py

```diff
@@ -1285,6 +1293,42 @@ def test_manifest_filter_validation(self):
         response = requests.get(url)
         self.assertEqual(400, response.status_code, response.content)
 
+    @mock.patch('azul.service.manifest_service.ManifestService.get_cached_manifest')
+    def test_fetch_manifest(self, get_cached_manifest):
```

Comment:
This sticks out in that it doesn't follow the pattern of the rest of the methods in this class. It doesn't use the `manifest_test` decorator or depends on the fixtures this class provides.

I like the fact that this covers code in `app.py` but it would be nice if this covered /manifest/files and the cache-miss code paths there as well. I think we should move this into its own class, and add a fixme plus ticket for the additional coverage.

Reply:
Created #2414